about.html

```diff
@@ -12,62 +14,82 @@
-        
-      <footer>
+  <div>
+    <header>
+        <img src="img/lambda-black.png" alt="lambda-Logo">
         <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
+            <a href="index.html">Home</a>
+            <a href="">About</a>
+            <a href="">Products</a>
+            <a href="">Blog</a>
+            <a href="">Contact</a>
         </nav>
-      </footer>
-    </div><!-- container -->        
+    </header>
+    <div class='under-header-img'>
+        <img src="img/jumbo-about.png" alt="Jumbo-UnderHeader-img">
+    </div>
+  </div>
+    
+  <div class="container about-page">
+    <section class="about-section">
+      <h2>About</h2>
+
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate
+        strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world
+        view of disruptive innovation via workplace diversity and empowerment.</p>
+    </section>
+
+    <section class="middle-top">
+      <div class="quick-info">
+        <img src="img/about-plan.png" alt="strategy">
+        <h3>Strategy</h3>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button>Learn More</button>
+      </div>
+
+      <div class="quick-info">
+        <img src="img/about-working.png" alt="strategy">
+        <h3>How We Work</h3>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button>Learn More</button>
+      </div>
+
+    </section>
+
+    <section class="middle-bottom">
+      <div class="quick-info">
+        <img src="img/about-office.png" alt="strategy">
+        <h3>Places We Work</h3>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button>Learn More</button>
+      </div>
+
+      <div class="quick-info">
+        <img src="img/about-meeting.png" alt="strategy">
+        <h3>Collaboration</h3>
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+        <button>Learn More</button>
+      </div>
+
+    </section>
+
+    <div class="pre-footer-content">
+      <h4>Let's Work Together</h4>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate
+      strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world
+      view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
```

Comment:
You could be a little more semantic in places (i.e. using `section`s over `div`s for a lot of these smaller sections):
```html
<section class="pre-footer-content">
  <h4>Let's Work Together</h4>
  <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. 
    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
    Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
</section>
```